pkg/kubectl/cmd/drain.go

```diff
@@ -495,7 +495,9 @@ func (o *DrainOptions) evictPods(pods []api.Pod, policyGroupVersion string, getP
 				if err == nil {
 					break
 				} else if apierrors.IsTooManyRequests(err) {
-					time.Sleep(5 * time.Second)
+					retryInterval := 5
```

Comment:
@gyliu513 : In my understanding (refer link below, look at item 2)
https://kubernetes.io/docs/tasks/administer-cluster/configure-pod-disruption-budget/
err = o.evictPod(pod, policyGroupVersion)
which invokes /eviction API has only 3 type of responses.
err code 429 is due to PDB